package/network/services/openvpn/files/openvpn.init

```diff
@@ -130,3 +135,56 @@ start_service() {
 service_triggers() {
 	procd_add_reload_trigger openvpn
 }
+
+LOG="logger -t openvpn-prepare -p"
+prepare() {
+	local s="$1"
+
+	local log="logger -t openvpn-prepare -p"
+
+	[ -z "$s" ] && {
+		$log error "Config not specified"
+		exit 1
+	}
+
+	uci -q get openvpn.$s > /dev/null || {
+		$log error "Config $s not found"
+		exit 2
+	}
+
+	. /usr/share/openvpn/openvpn.options
+
+	config_load 'openvpn'
+	config_get enabled "$s" 'enabled' 0
```

Comment:
It looks like the current `/etc/init.d/openvpn` uses either `enable` or `enabled`. I still had `enable` in my config files, probably from the older OpenWRT days.